docs/api-docs/store-logs/store-logs.md

```diff
@@ -0,0 +1,329 @@
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/store/systemlogs?module=order+status
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example response: Filter by module" lineNumbers
+{
+  "data": [
+    {
+      "id": 2,
+      "type": "ordersettings",
+      "module": "Order Status",
+      "severity": "success",
+      "summary": "Successfully customized order status",
+      "message": "{\"id\":11,\"customLabelBefore\":\"Awaiting Shipment\",\"customLabelAfter\":\"Package ready to be picked up by Fed Ex\",\"userId\":4,\"userUid\":3172923,\"userEmail\":\"user@example.com\",\"userRole\":\"admin\"}",
+      "date_created": "2021-12-18T04:46:40+00:00"
+    }
+  ],
+  "meta": {
+    "pagination": {
+      "total": 1,
+      "count": 1,
+      "per_page": 50,
+      "current_page": 1,
+      "total_pages": 1,
+      "links": {
+        "current": "?module=order+status&page=1&limit=50"
+      }
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+
+### Filter by severity 
+
+Each log entry describes an event that has an associated outcome, which this API refers to as its `severity`. To filter the results by severity, use the `severity` query parameter and specify the severity's corresponding integer in the query. For a list of severity values, see the [Get system logs](/api-reference/store-management/store-logs/system-logs/get-system-logs) endpoint.
+
+<!--
+type: tab
+title: Request
+-->
+
+```http title="Example request: Filter by severity" lineNumbers
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/store/systemlogs?severity=2
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example response: Filter by severity" lineNumbers
+{
+  "data": [
+    {
+      "id": 3,
+      "type": "general",
+      "module": "",
+      "severity": "notices",
+      "summary": "Marking order 1234000002 as deleted.",
+      "message": "Marking order 1234000002 as deleted.",
+      "date_created": "2021-12-19T19:12:31+00:00"
+    },
+    {
+      "id": 4,
+      "type": "general",
+      "module": "",
+      "severity": "notices",
+      "summary": "Restoring order 1234000002",
+      "message": "Restoring order 1234000002",
+      "date_created": "2021-12-19T19:15:56+00:00"
+    }
+  ],
+  "meta": {
+    "pagination": {
+      "total": 2,
+      "count": 2,
+      "per_page": 50,
+      "current_page": 1,
+      "total_pages": 1,
+      "links": {
+        "current": "?severity=2&page=1&limit=50"
+        }
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+### Filter by paginated results   
+
+To specify the page and number of entries returned per page, use the query parameters `page` and `limit`, respectively. The following example shows a response that starts at entry 4, as you request page 2 with 3 entries per page.  
```

Comment:
maybe rewrite to say: "which starts at entry 4 when you submit a request for page 2 with 3 entries per page."

Reply:
rewritten!